web/src/main/java/org/mskcc/cbio/oncokb/controller/EvidenceController.java

```diff
@@ -222,6 +222,27 @@ ResponseEntity updateVUS(@ApiParam(value = "hugoSymbol", required = true) @PathV
         Set<ClinicalTrial> clinicalTrials = queryEvidence.getClinicalTrials();
         String propagation = queryEvidence.getPropagation();
         
+        List<String> cancerTypes = new ArrayList<>();
+        List<String> subTypes = new ArrayList<>();     
+        Boolean cancerEvidence = true;
```

Comment:
As I mentioned before, can we make the variable name more clear here? `isCancerEvidence` would be easier to understand.